### PR TITLE
CI: don't require to have parsers installed to check for injections

### PR DESCRIPTION
**Pull request by stsewd**
A faster fix for https://github.com/nvim-treesitter/nvim-treesitter/pull/1651

### CODE REVIEW

**Review by theHamsta — APPROVED** [review]

File: scripts/check-queries.lua [hunk 1]
```diff
@@ -18,7 +18,7 @@ local function extract_captures()
   end
 
   -- Complete captures for injections.
-  local parsers = require("nvim-treesitter.info").installed_parsers()
+  local parsers = require("nvim-treesitter.parsers").list
   for _, lang in pairs(parsers) do
```
Comment (theHamsta) [inline scripts/check-queries.lua:22]: ipairs

Comment (theHamsta) [reply scripts/check-queries.lua:22]: and `lang`, _  lang is the key

Comment (theHamsta) [reply scripts/check-queries.lua:22]: or just vim.tbl_keys

Comment (stsewd) [reply scripts/check-queries.lua:22]: just pushed a change to use tbl_keys p:

**Pull request merged**